libsolidity/interface/CompilerStack.cpp

```diff
@@ -100,6 +105,13 @@ bool CompilerStack::parse()
 	m_errors.clear();
 	m_parseSuccessful = false;
 
+	if (SemVerVersion{string(VersionString)}.isPrerelease())
+	{
+		auto err = make_shared<Error>(Error::Type::Warning);
```

Comment:
@chriseth For some reason this gets detected as an error by browser-solidity as it refuses to deploy the contract.